packages/@lwc/babel-plugin-component/src/__tests__/decorators.spec.js

```diff
@@ -194,3 +194,77 @@ describe('decorators', () => {
         }
     );
 });
+
+describe('transform large input file', () => {
+    beforeEach(() => {
+        // Disable console.error because babel logs warnings if the file size is > 500KB
+        const { error: originalConsoleError } = console;
+        // eslint-disable-next-line no-console
+        console.error = jest.spyOn(console, 'error').mockImplementation((...args) => {
+            if (
+                args &&
```

Comment:
When using arguments spread, the spread always returns an array even when no arguments are passed.

```js
const log = (...args) => console.log(Array.isArray(args), args);

log(); // true []
log.call(null); // true []
log.apply(null, []); // true []
```

```
```

```suggestion
```